source/patterns/04-pages/blog-post/blog-post.twig

```diff
@@ -1,4 +1,15 @@
 {% include 'templates-page-grid' with {
+  start: {
+    template: 'templates-content-grid',
```

Comment:
Suspect we might want the site header to be a grid itself.

Reply:
Probably true. Might have the grid self-contained within the site header pattern rather than as a separate grid pattern though. Can update `start` if necessary.